superset/examples/utils.py

```diff
@@ -44,28 +44,30 @@ def load_examples_from_configs(
 
 def load_contents(load_test_data: bool = False) -> dict[str, Any]:
     """Traverse configs directory and load contents"""
-    root = Path("examples/configs")
-    resource_names = resource_listdir("superset", str(root))
-    queue = [root / resource_name for resource_name in resource_names]
+    root = files("superset") / "examples/configs"
+    resource_names = (files("superset") / str(root)).iterdir()
+    queue = [root / str(resource_name) for resource_name in resource_names]
 
     contents: dict[Path, str] = {}
     while queue:
         path_name = queue.pop()
         test_re = re.compile(r"\.test\.|metadata\.yaml$")
 
-        if resource_isdir("superset", str(path_name)):
+        if (files("superset") / str(path_name)).is_dir():
             queue.extend(
-                path_name / child_name
-                for child_name in resource_listdir("superset", str(path_name))
+                path_name / str(child_name)
+                for child_name in (files("superset") / str(path_name)).iterdir()
             )
-        elif path_name.suffix.lower() in YAML_EXTENSIONS:
+        elif Path(str(path_name)).suffix.lower() in YAML_EXTENSIONS:
```

Comment:
Same comment as previously.

Reply:
This is just a straight forward explicit coercion from Traversable -> str -> Path in order then call the `Path.suffix()` method.